### ISSUE TITLE
Animania entity names or IDs?

### ISSUE THREAD

**Issue by PurpleHel** [capnkirok/animania#335]
Maybe I'm just completely failing, but how do I teleport animania animals? I can't find the right entity names to use with /tp @e[type=WHATEVER] @p, or @e[name=WHATEVER]and when I try the IDs listed in the config for /tp @e[id=WHATEVER] @p I get all the entities in range. (r=5 is so great when trying to find entity types/names.)

I just want to teleport all my cows into their barn, and all my chickens into their coop. I'm playing on my own server that only my friends and I use, and I use the range command when teleporting, so me teleporting things around won't ruin anyone else's fun. :)

**Comment by Tschipp** [comment]
Which version are you using?

**Comment by capnkirok** [comment]
What version are you running? (1.10 - 1.12)?

Also, have you tried Carry On yet for moving animals? Our companion mod?

**Comment by PurpleHel** [comment]
Sorry, derp. 1.10, and the latest animania, 1.3.1a

And no, I haven't tried Carry On. I've already got multiple options for transporting single animals at a time-golden lasso, piggybackpack. It is cool that it allows you to move chests and stuff, and it may well replace Chest Transporter for me. (in fact, no, it totally *is* going to replace chest transporter next time I reboot my server.) But I don't want to move all my animals one at a time. My available time to actually play minecraft is limited, and moving animals 1 by 1 is, for me, a terrible use of my time.

I want to move every single Holstein cow, then every bull, then every calf, with a single command each. I just don't know what to put in for 'WHATEVER' in tp @e[type=WHATEVER] @p. For example, if I want every vanilla horse, it's type=EntityHorse. If I want every ocelot/tamed cat, it's type=ozelot in 1.10, and =ocelot in 1.12. But if I want every white peahen? Every Holstein cow? I got nothing. I've tried to guess it based on crash logs that came up in google, but no luck.
